catalog/transformer/catalog-transformer-rtf/src/test/java/ddf/catalog/transformer/output/rtf/RtfQueryResponseAndMetacardTransformerTest.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.output.rtf;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import ddf.catalog.data.BinaryContent;
+import ddf.catalog.data.Metacard;
+import ddf.catalog.data.Result;
+import ddf.catalog.operation.SourceResponse;
+import ddf.catalog.transform.CatalogTransformerException;
+import ddf.catalog.transformer.output.rtf.model.RtfCategory;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import org.junit.Before;
+import org.junit.Test;
+
+public class RtfQueryResponseAndMetacardTransformerTest extends BaseTestConfiguration {
+
+  private List<RtfCategory> mockCategories;
+
+  @Before
+  public void setup() {
+    mockCategories = getCategories();
+  }
+
+  @Test
+  public void testCreateTransformer() {
+    RtfQueryResponseAndMetacardTransformer transformer = createTransformer();
+
+    assertThat("Transformer cannot be null", transformer, notNullValue());
+  }
+
+  @Test(expected = CatalogTransformerException.class)
+  public void testTransformNullMetacard() throws CatalogTransformerException {
+    RtfQueryResponseAndMetacardTransformer transformer = createTransformer();
+
+    assertThat("Transformer cannot be null", transformer, notNullValue());
```

Comment:
❓ In a test that expects an exception should any assertions be made?

Reply:
Yes, because this is not the place I would be expecting an exception 😄